# HG changeset patch
# User Rik <rik@octave.org>
# Date 1390595313 28800
#      Fri Jan 24 12:28:33 2014 -0800
# Node ID e33f706468bc8648aa4f3fda0383f9e92f5878dd
# Parent  12eb4eb90486e646da1908648f3be97ccde8c66e
Fix extra space in dbwhere output.

* debug.cc (Fdbwhere): Remove extra space between "at" and "line".

diff --git a/libinterp/corefcn/debug.cc b/libinterp/corefcn/debug.cc
--- a/libinterp/corefcn/debug.cc
+++ b/libinterp/corefcn/debug.cc
@@ -839,28 +839,28 @@ execution is stopped.\n\
         }
 
       octave_stdout << "stopped in " << name << " at ";
 
       int l = octave_call_stack::caller_user_code_line ();
 
       if (l > 0)
         {
-          octave_stdout << " line " << l << std::endl;
+          octave_stdout << "line " << l << std::endl;
 
           if (have_file)
             {
               std::string line = get_file_line (name, l);
 
               if (! line.empty ())
                 octave_stdout << l << ": " << line << std::endl;
             }
         }
       else
-        octave_stdout << " <unknown line>" << std::endl;
+        octave_stdout << "<unknown line>" << std::endl;
     }
   else
     error ("dbwhere: must be inside a user function to use dbwhere\n");
 
   return retval;
 }
 
 // Copied and modified from the do_type command in help.cc
